.mise/tasks/infrastructure/kubeflow/build

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+
+# mise description="build docker image for kubeflow infrastructure"
+# mise tools=["docker"]
+
+cd $root/infrastructure/kubeflow
+IMAGE=pocketsizefund/kubeflow
+
+docker build --build-arg PULUMI_ACCESS_TOKEN=$PULUMI_ACCESS_TOKEN -t $IMAGE:latest --platform $PLATFORM .
```

Comment:
Nitpick but I like separating longer commands into multiline format for easier reading.

Reply:
works for me, fixed